test/remote/gateways/remote_multi_pay_test.rb

```diff
@@ -0,0 +1,76 @@
+require 'test_helper'
+
+class RemoteMultiPayTest < Test::Unit::TestCase
+  def setup
+    @gateway = MultiPayGateway.new(fixtures(:multi_pay))
+
+    @amount = 100
```

Comment:
Might be helpful to include this value in a test to assert that it is what we expect it should be (`1.00`)

Reply:
yes, I will include a test for amount